backend/migrations/versions/4c9e3982a64b_create_user_theatrical_role_.py

```diff
@@ -0,0 +1,45 @@
+"""create user_theatrical_role_relationship table
+
+Revision ID: 4c9e3982a64b
+Revises: 4ef6789fa0fc
+Create Date: 2023-12-16 01:46:26.170771
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = "4c9e3982a64b"
+down_revision: Union[str, None] = "4ef6789fa0fc"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
```

Comment:
Do we need typing here?
The thing is that we're declaring variables here and it's type defined by us (developers). So if you write
`my_string = "asdasd"` it will be str type. Same for other variables/types. For me it's kind of redundant action